csrc/quantization/cutlass_w8a8/scaled_mm_c2x.cu

```diff
@@ -125,6 +125,46 @@ struct ScaledEpilogue {
   }
 };
 
+template <typename ElementD, typename OutputTileThreadMap>
+struct ScaledEpilogueAzp
+    : private ScaledEpilogue<ElementD, OutputTileThreadMap> {
+ private:
+  using SUPER = ScaledEpilogue<ElementD, OutputTileThreadMap>;
+  using ScaleA = typename SUPER::ScaleA;
+  using ScaleB = typename SUPER::ScaleB;
+  using EVTCompute0 = typename SUPER::EVTCompute0;
+
+  using Compute1 = cutlass::epilogue::threadblock::VisitorCompute<
+      cutlass::multiply_add, ElementD, float,
+      cutlass::FloatRoundStyle::round_to_nearest>;
+
+  using Bias = cutlass::epilogue::threadblock::VisitorColOrScalarBroadcast<
+      OutputTileThreadMap, float, Stride<Int<1>, Int<0>, Int<0>>>;
```

Comment:
I don't think there will ever be a usecase for a scalar bias, so I think it will be better to use`VisitorColBroadcast`, which is already in CUTLASS